R/pkg/inst/extdata/spark_download.csv

```diff
@@ -0,0 +1,2 @@
+"url","default"
+"http://apache.osuosl.org",TRUE
```

Comment:
I don't see the strong reason for a CSV file listing available mirrors. Because we can support apache dynamic mirrors, and install_spark() has a parameter allowing user to pass a vector of mirror sites.